main.go

```diff
@@ -11,45 +11,44 @@ import (
 )
 
 func main() {
-	err := run(os.Args)
+	message, err := run(os.Args)
+	if message != "" {
+		fmt.Println(message)
+	}
```

Comment:
Previously message was always printed regardless if it was empty or not. Any specific reason for the change?

Reply:
I don't think message was normally empty before, but now `run()` regularly return "" on error, meaning the `if` below will print instead.